datarobot_batch_scoring/batch_scoring.py

```diff
@@ -134,7 +137,7 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                     [c for c in keep_cols if c not in first_row.fieldnames],
                     first_row.fieldnames))
 
-        if not dry_run:
+        if not (dry_run or import_id):
```

Comment:
we don't use authorization for SSE. Neither user/password nor /user/token, so this part of logic could be skipped as it would be a dry-run call